frontend/src/metabase/home/components/Onboarding/OnboardingDismissedToast.tsx

```diff
@@ -0,0 +1,16 @@
+import { t } from "ttag";
+
+import { Stack, Text, Title } from "metabase/ui";
+
+export const OnboardingDismissedToast = () => {
```

Comment:
nit, not a blocker: ![image](https://github.com/user-attachments/assets/90854516-2e44-4446-aaf1-d309ece56160)

The spacing here feels a bit off

Reply:
Off in what sense (or in which direction)? Vertically/horizontally?

This should (hopefully) be a temporary thing.
@romnitt hopes we will start using Mantine notifications soon-ish because this toast needs to live in the upper right corner (custom position), and to have completely custom styling.

To the best of my knowledge, none of our custom toast implementations support both at the moment.

Reply:
Oh sorry, I mean the spacing inside the toast - between the icon, heading and subheading. Sounds like this conforms to Figma so all is good.

Reply:
Nothing to be sorry for.
I appreciate the thorough review and your attention to detail!

I just really really really hope we can start using Mantine notifications soon. Feels silly to have a bunch of custom implementations that are not flexible enough to implement a fairly simple design requirement.